packages/admin/cms-admin/src/common/routeMenu/getRoutesFromRouteMenu.ts

```diff
@@ -0,0 +1,15 @@
+import { RouteProps } from "react-router-dom";
+
+import { RouteMenu, RouteMenuItemWithSubMenu } from "./routeMenu.type";
+
+export function getRoutesFromRouteMenu(items: RouteMenu<10>): RouteProps[] {
```

Comment:
:+1: i like the new function.

however the naming.... it doesn't return Routes, it returns RouteProps?